### ISSUE TITLE
something wrong with predictions  Cannot load image "data/labels/126_7.png"

### ISSUE THREAD

**Issue by AbhimanyuAryan** [pjreddie/darknet#1759]
while training the model I was getting messages like this

```
...
Region 16 Avg IOU: -nan, Class: -nan, Obj: -nan, No Obj: 0.002315, .5R: -nan, .75R: -nan,  count: 0
Region 23 Avg IOU: 0.603194, Class: 0.618950, Obj: 0.259226, No Obj: 0.001301, .5R: 0.812500, .75R: 0.187500,  count: 16
648: 3.516367, 3.871164 avg, 0.000176 rate, 0.029526 seconds, 2592 images
Loaded: 0.022205 seconds
...
```

**I followed this guide to from labelling(VOTT) to training**: https://qiita.com/PINTO/items/7dd7135085a7249bf17a

command for prediction: 

```
./darknet detector test data/obj.data cfg/yolov3-tiny-mine.cfg backup/yolov3-tiny-mine_700.weights data/test/ir_pred.jpg

```

My data directory

![1](https://user-images.githubusercontent.com/8083613/62541054-9d868000-b876-11e9-8e9c-d164863fce93.png)

**obj.data**
```
classes = 1
train  = data/train.txt
valid  = data/test.txt
names = data/obj.names
backup = backup/
```

**yolov3-tiny-mine.cfg**
```
[net]
# Testing
#batch=1
#subdivisions=1
# Training
batch=4
subdivisions=1
width=416
height=416
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.001
burn_in=1000
max_batches = 500200
policy=steps
steps=400000,450000
scales=.1,.1

[convolutional]
batch_normalize=1
filters=16
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=32
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=64
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=128
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=1

[convolutional]
batch_normalize=1
filters=1024
size=3
stride=1
pad=1
activation=leaky

###########

[convolutional]
batch_normalize=1
filters=256
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[convolutional]
#filters=(classes+5)*3
size=1
stride=1
pad=1
filters=18
activation=linear



[yolo]
mask = 3,4,5
anchors = 10,14,  23,27,  37,58,  81,82,  135,169,  344,319
classes=1
num=6
jitter=.3
ignore_thresh = .7
truth_thresh = 1
random=1

[route]
layers = -4

[convolutional]
batch_normalize=1
filters=128
size=1
stride=1
pad=1
activation=leaky

[upsample]
stride=2

[route]
layers = -1, 8

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[convolutional]
#filters=(classes+5)*3
size=1
stride=1
pad=1
filters=18
activation=linear

[yolo]
mask = 0,1,2
anchors = 10,14,  23,27,  37,58,  81,82,  135,169,  344,319
classes=1
num=6
jitter=.3
ignore_thresh = .7
truth_thresh = 1
random=1
```

**Comment by AbhimanyuAryan** [comment]
fixed

**Comment by Jonashellevang** [comment]
> fixed

Hey! How did you solve this issue? I am having the same one right now. Training works, but testing doesn't. How did you solve this?
